tests/python/relay/dyn/test_dynamic_op_level3.py

```diff
@@ -202,5 +209,152 @@ def verify_sparse_to_dense(sparse_indices, sparse_values, default_value, output_
+                current_idx += 1
+
+            return current_idx
+
+        current_idx = 0
+        new_sparse_indices = []
+        new_sparse_values = []
+        empty_row_indicator = [0 for _ in range(dense_shape[0])]
+        num_cols = sparse_indices.shape[1]
+        for sparse_row, sparse_value in zip(sparse_indices, sparse_values):
+            limit_idx = sparse_row[0]
+            current_idx = check_add_rows(current_idx, limit_idx)
+            new_sparse_indices.append(list(sparse_row))
+            new_sparse_values.append(sparse_value)
+            current_idx = limit_idx + 1
+
+        check_add_rows(current_idx, dense_shape[0])
+        return new_sparse_indices, new_sparse_values, empty_row_indicator
+
+    def verify_sparse_fill_empty_rows(
+        sparse_indices_np: np.ndarray,
+        sparse_values_np: np.ndarray,
+        dense_shape_np: np.ndarray,
+        default_value_np: np.ndarray,
+    ) -> None:
+        """
+        This function verifies the relay output of sparse_fill_empty_rows with its expected output.
+        """
+        if use_dyn:
+            sparse_indices = relay.var(
+                "sparse_indices",
+                shape=[relay.Any(), relay.Any()],
+                dtype=str(sparse_indices_np.dtype),
+            )
+            sparse_values = relay.var(
+                "sparse_values",
+                shape=[relay.Any()],
+                dtype=str(sparse_values_np.dtype),
+            )
+            dense_shape = relay.var(
+                "dense_shape",
+                shape=[relay.Any()],
+                dtype=str(dense_shape_np.dtype),
+            )
+            default_value = relay.var(
+                "default_value",
+                shape=[relay.Any()],
+                dtype=str(default_value_np.dtype),
+            )
+        else:
+            sparse_indices = relay.var(
+                "sparse_indices",
+                relay.TensorType(sparse_indices_np.shape, str(sparse_indices_np.dtype)),
+            )
+            sparse_values = relay.var(
+                "sparse_values",
+                relay.TensorType(sparse_values_np.shape, str(sparse_values_np.dtype)),
+            )
+            dense_shape = relay.var(
+                "dense_shape",
+                relay.TensorType(dense_shape_np.shape, str(dense_shape_np.dtype)),
+            )
+            default_value = relay.var(
+                "default_value",
+                relay.TensorType(default_value_np.shape, str(default_value_np.dtype)),
+            )
+        z = relay.sparse_fill_empty_rows(
+            sparse_indices, sparse_values, dense_shape, default_value
+        ).astuple()
+        func = relay.Function([sparse_indices, sparse_values, dense_shape, default_value], z)
+        ref_res = ref_sparse_fill_empty_rows(
+            sparse_indices_np,
+            sparse_values_np,
+            dense_shape_np,
+            default_value_np,
+        )
+        verify_func(
```

Comment:
Can you also verify that the output dtype is equal to the input dtype?

Reply:
I have set the output dtype to int64 irrespective of input . Is this an issue ?

Reply:
I actually don't know how to store write int32 / int64 in a parametrized way in hybrid script.

Reply:
The output type should match the input type.

Reply:
Done.